packages/core/src/index.js

```diff
@@ -1,6 +1,8 @@
 import './global/js/polyfills';
 import boot from './global/js/boot';
+import settingsObj from './global/js/settings';
 
 boot();
 
 export * from './components';
+export const settings = settingsObj;
```

Comment:
```suggestion
export { default as settings } from './global/js/settings';
```

Reply:
I reject your suggestion and commit it myself

Reply:
:(